src/App/ComplexGeoData.cpp

```diff
@@ -184,6 +184,12 @@ bool ComplexGeoData::getCenterOfGravity(Base::Vector3d& unused) const
     return false;
 }
 
+bool ComplexGeoData::getRotation(Base::Rotation& unused) const

```

Comment:
Calling the method getRotation() is problematic because it's assumed that it's related to the placement API of that class, but it's not.

Reply:
I removed the getRotation() method and added a method in GeoFeature that provides a direction instead of a rotation.